hw/lint/Makefile

```diff
@@ -92,12 +93,31 @@ $(ips_vlint):
 	rm -rf build
 	mkdir -p ${REPORT_DIR}
 	-fusesoc --cores-root ${CORE_ROOT} run --target=lint lowrisc:$(subst -,:,$(patsubst %_vlint,%,$@))
-#	cp build/lowrisc_*$(patsubst %_vlint,%,$@)*/vlint-verilator/verilator-lint.rpt ${REPORT_DIR}/$@.rpt
 
-# TODO: add a verilator summary report function
-# add a summary report option for verilator
+# TODO: add summary reporting
 vreport:
 
+##############################
+# verible style lint targets #
+##############################
+
+sall: slint
```

Comment:
Would the user ever want to run lint without generating the reports? If no, then I will recommend removing `vall` and `sall`.  

Also, I recommend `all: lint` to run verilog lint by default if the user just runs `make`.